open5gcore_vm/changelog.md

```diff
@@ -5,6 +5,12 @@
 ### Added
```

Comment:
Please add a version number. Will be use to track this component in the global version.